jsf/plugins/org.jboss.tools.jsf.ui/src/org/jboss/tools/jsf/ui/operation/JSFProjectCreationOperation.java

```diff
@@ -86,6 +86,7 @@ protected void configFacets(IDataModel dataModel, String projectLocation) {
 					: template.indexOf("2.0") >= 0 ? "2.0"
 					: template.indexOf("2.1") >= 0 ? "2.1"
 					: template.indexOf("2.2") >= 0 ? "2.2"
+					: template.indexOf("2.2") >= 0 ? "2.3"

```

Comment:
Shouldn't it be template.indexOf("2.3") ?

Reply:
Thanks!

Reply:
Fixed